service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsNotificationServiceTest.java

```diff
@@ -72,7 +79,7 @@
 
     @BeforeEach
     void setup() {
-        Logger logger = (Logger) getLogger(RoboticsDataVerificationHelper.class.getName());
+        Logger logger = (Logger) getLogger(RoboticsLoggerHelper.class.getName());
```

Comment:
conceptually it should be in BeforeAll as logger is static filed of class and executing it multiple class adds multiple appenders to logger